homeassistant/components/pvpc_hourly_pricing/translations/en.json

```diff
@@ -19,8 +19,7 @@
             "init": {
                 "data": {
                     "power": "Contracted power (kW)",
-                    "power_p3": "Contracted power for valley period P3 (kW)",
-                    "tariff": "Applicable tariff by geographic zone"
+                    "power_p3": "Contracted power for valley period P3 (kW)"
```

Comment:
May be you could link the translations of the option flow to those of the config flow. (can also be done in a separate PR).

Reply:
That sounds good! Could you point me to some example of that? 🙏

Reply:
Oh, I think I see what you mean, something like `"description": "[%key:component::purpleair::config::step::by_coordinates::description%]",`, right? 👀

I'll keep that in mind for the next iteration, thanks 👌

Reply:
> can also be done in a separate PR

or maybe push here one more 2-LOC commit with the change for those 2 repeated keys? 

both are ok with me 👍

```json
          "power": "[%key:component::pvpc_hourly_pricing::config::step::user::data::power%]",
          "power_p3": "[%key:component::pvpc_hourly_pricing::config::step::user::data::power_p3%]"
```

Reply:
Right, I did a similar job with the MQTT localization strings. This saves time for the contributors that do the translations.

Reply:
Go ahead if you like to additional do commits for the translations.